myanimelist.py

```diff
@@ -0,0 +1,72 @@
+from bs4 import BeautifulSoup
+import requests
+import json
+
+
+json_file=open('anime_scrape.json','w')
+
+#Function to get url and return the data to scrape from it
+def make_soup(url):
+	source=requests.get(url)
+	soupdata=BeautifulSoup(source.content,'lxml')
+	return soupdata
+
+'''
+Function to store the major information about a anime
+and storing the data in anime_scrape.json file
+'''
+
+def info_anime(soup):
+
+	#Extracting the name of the anime
+
+	anime=soup.find(name="span",attrs={"itemprop":"name"})
+	name=anime.text
+	print ("Anime : "+name)
+	json.dump(name,json_file)
+
+
+
+	#Extracting the rating 
+
+	rating=soup.find(name="div",attrs={"class":"fl-l score"})
+	print ("Rating : "+(rating.text.strip()))
+	json.dump(rating.text.strip(),json_file)
+
+
+	#extracting the description
+
+	des=soup.find(name="span",attrs={"itemprop":"description"})
+	description=des.text
+	print ("Description : "+description)
+	json.dump(description,json_file)
+
+	#Extracting the Rank
+
+	rank=soup.find(name="span",attrs={"class":"numbers ranked"})
+	print (rank.text)
+
+	#Extracting number of episodes
+
+	ep=soup.find(name="div",attrs={"class":"spaceit"})
+	print (ep.text)
+
+
+if __name__ == '__main__':
+	'''
+	Iterating the loop from 1 to 30000 to store data from myanimelist
+	where the url is  https://myanimelist.net/anime/(some number)/
+	'''
+
+for index in range(1,30000):
```

Comment:
Indentation required

Reply:
![screenshot from 2018-12-09 16-56-22](https://user-images.githubusercontent.com/31985436/49696697-bcdbe900-fbd3-11e8-80f8-a4dc66465580.png)
I don't see any indentation issue in my code, rather the code in github is showing up differently.